src/seeds/seeder.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { Users } from '../users/users.model';
+import { NewUser } from '../types/users';
+
+@Injectable()
+export class SeederRepository {
+  constructor(
+    @Inject(Users.name)
+    private readonly userModel: typeof Users,
+  ) {}
+
+  checkDataUsers(id: number): Promise<boolean> {
```

Comment:
can be renamed to isExist or something like that.

Reply:
fixed